blocks/api/serializer.js

```diff
@@ -64,6 +63,36 @@ export function getCommentAttributes( realAttributes, expectedAttributes ) {
 }
 
 /**
+ * Takes block details and returns the serialized content.
+ *
+ * @param {String} blockType			The type of block, e.g. 'core/embed'
+ * @param {Object} blockAttributes		The block's attributes
+ * @param {Object} blockSettings		The block's settings object
+ * @param {String} saveContent			Content to save into the post
+ * @return {String}						The post content
+ */
+export function serializeBlock( blockType, blockAttributes, blockSettings, saveContent ) {
```

Comment:
instead of taking all these arguments, should we replace those with a block object instead?